spec/dummy/config/initializers/secret_token.rb

```diff
@@ -1,7 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-Dummy::Application.config.secret_token = 'aa65efdc74ffe579a42d95ab0d9f6a47305420e30e92f6ea1791491909b6397801261700300b15a1dae477328a2ebf7fe426afb43aa58544bf616ee42a6cb255'
```

Comment:
I don't think that's necessary, it's only in the dummy config and `secret_token` hasn't been removed since rails 6 after it was replaced with `secret_key_base` in rails 4, so it hasn't even been being used for some time now
And also seems like if you don't specify one rails generates either a `developer_secrets.txt` or `local_secrets.txt` for local use

Reply:
If Rails generates something on the fly at this point, than this comment and all others can just be resolved.

Reply:
Didn't realize there was another thread here; responded over here: https://github.com/rapid7/metasploit-credential/pull/181#discussion_r1656814535